### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by delbaoliveira**
Day 1: Using JSON web tokens

### CODE REVIEW

**Comment by delbaoliveira** [comment]
Day 2: Client side authentication

File: app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  };
+
+  const handleLogin = token => {
+    localStorage.setItem("jwt", token);
+    setLoggedStatus(true);
+  };
+
+  // When the component first renders
+  // Check if the token exists in local storage (if user previously signed in) and update state
+  useEffect(() => {
```
Comment (Velsu) [inline app/src/App.js]: This is really nice piece of functionality 👍

File: app/src/components/Signin.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+const Signin = props => {
+  const [username, setUsername] = useState("");
```
Comment (Velsu) [inline app/src/components/Signin.js]: Great usage of React Hooks 🔥

File: index.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      }
+    })
+    .catch(error => {
+      res.status(500).json(error);
+    });
+});
+
+// GET - users if logged in ['/api/users']
+// 4. Verify the token sent by the client (through a middleware) in the authorization header
+function tokenVerifier(req, res, next) {
```
Comment (Velsu) [inline index.js]: I believe You could make Your code a little more clean and modular if You delegated all middleware and web token methods into the separate file and imported them to routes.

**Pull request merged**